packages/graphql-codegen/src/generateEnumsGraphql.ts

```diff
@@ -1,14 +1,21 @@
 import { camelCase } from "change-case";
 import { CodeGenFile, EnumMetadata } from "joist-codegen";
-import { formatGraphQL, mapTypescriptTypeToGraphQLType } from "./graphqlUtils";
+import { formatGraphQL, isJsonbColumn, mapTypescriptTypeToGraphQLType, SupportedTypescriptTypes } from "./graphqlUtils";
 
 /** Generates a `schema/enums.graphql` with GQL enums that match all of our domain enums. */
 export async function generateEnumsGraphql(enums: EnumMetadata): Promise<CodeGenFile> {
   const contents = Object.values(enums)
     .map(({ name, rows, extraPrimitives }) => {
       const enumDecl = `enum ${name} { ${rows.map((r) => r.code).join(" ")} }`;
       const detailDecl = `type ${name}Detail { code: ${name}! name: String! ${extraPrimitives
-        .map((p) => `${camelCase(p.columnName)}: ${mapTypescriptTypeToGraphQLType(p.fieldName, p.fieldType)}!`)
+        .filter((p) => !isJsonbColumn(p))
```

Comment:
For now just not supporting `jsonb` columns as extra columns on primitives... Should be doable in a future PR.

Reply:
Specifically in the GQL output...I'm not actually sure what they'd look like in the domain model, they might work-ish? Not sure.